CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+cmake_minimum_required(VERSION 3.9)
+project(Algorithms_in_C++
+    LANGUAGES CXX
+    VERSION 1.0.0
+    DESCRIPTION "Set of algorithms implemented in C++."
+)
+
+# set(CMAKE_CXX_CPPLINT "~/anaconda3/bin/cpplint --filter=-legal/copyright --std=c++11")
+# find_program(CLANG_FORMAT "clang-format")
+
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
why we are not using C++17

Reply:
to ensure the code can be compiled on older platforms and compilers as well - for example, people can use much of this work directly in IoT and wearable technologies.